README.md

```diff
@@ -1 +1,16 @@
-# enumerable
\ No newline at end of file
+# Enumerable
+This project is all about creating custom methods for enumerable module which have similar functionality to in-built one. This is second project of the Microverse program.
+
+# Languages used
+### Ruby programming language
+
+# How to install
+### https://github.com/sumancrest0001/enumerable.git
+### cd enumerable
+### Double click main.rb file to open it up in your text editor
+
+# Contributors
+### sumancrest0001 (https://github.com/sumancrest0001)
+
+# License
+MIT Licensed
```

Comment:
#### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), code samples of how it's used and what was learned during the project can make the project more descriptive.